LeaderboardBackend/Results.cs

```diff
@@ -9,7 +9,7 @@ namespace LeaderboardBackend.Result;
 public readonly record struct BadCredentials;
 public readonly record struct BadRole;
 public readonly record struct ConfirmationNotFound;
-public readonly record struct Conflict<T>(T Conflicting);
+public record Conflict<T>(T Conflicting);
```

Comment:
Why the change? .net doesn't like it if you keep it a read-only record struct?

Reply:
Doing it this way forces you to actually use the constructor, leading to fewer code errors.